src/backend/aspen/workflows/nextstrain_run/nextstrain_profile/aspen_auspice_config_v2.json

```diff
@@ -80,6 +80,10 @@
       "key": "submitting_lab",
       "title": "Submitting Lab",
       "type": "categorical"
+    },
+    {
+      "key": "gisaid_epi_isl",
+      "type": "categorical"
```

Comment:
this will port EPI_ISL info to the tree and get displayed in the sample tooltip.